src/_includes/macros/active-filters.html

```diff
@@ -0,0 +1,143 @@
+
+{# ==========================================================================
+
+   active_filters()
+
+   Description:
+
+   Create active filters markup when given:
+
+   filters:  An array of the filters that you want.
```

Comment:
Order of arguments in docs do not follow order in the macro render function definition on L:37.